Plugins/Bitbucket/BitbucketPullRequestForm.cs

```diff
@@ -275,9 +288,11 @@ private void UpdatePullRequestDescription()
                 (Commit)ddlBranchTarget.Tag,
                 _settings);
 
-            var result = getCommitsInBetween.Send();
+            var result = await getCommitsInBetween.SendAsync();
             if (result.Success)
             {
+                await this.SwitchToMainThreadAsync();
```

Comment:
We are on the main thread here. It can be invoked to ensure the control is not disposed. Perhaps it is better to check it explicitly (one can get tricked and remove this line as redundant).

Reply:
IMO it is not better to check it explicitly. (Or viewed a different way, this *is* the explicit check.)

1. This is going to a common pattern. A *really* common pattern.
2. This call is configured to work with JTF analyzers
3. This call "just works" regardless of whether the code starts on the UI thread and whether the control is still valid

Reply:
OK.